src/Protocols/Hedgehog.hs

```diff
@@ -60,6 +62,16 @@ resetGen n =
   C.unsafeFromActiveHigh
     (C.fromList (replicate n True <> repeat False))
 
+smallInt :: H.Range Int
+smallInt = Range.linear 0 10
+
+genSmallInt :: H.Gen Int
+genSmallInt =
+  Gen.frequency
+    [ (90, Gen.integral smallInt)
+    , (10, Gen.constant (Range.lowerBound 99 smallInt))
+    ]
+
```

Comment:
Consecutive stalling up to only 10 cycles is a very low default.
This should be configurable or derived from configuration? Could you make an issue and point to it here?

Reply:
Added! https://github.com/clash-lang/clash-protocols/issues/98